metrics/metrics.go

```diff
@@ -121,6 +124,8 @@ var (
 	VMApplyFlush                        = stats.Float64("vm/applyblocks_flush", "Time spent flushing vm state", stats.UnitMilliseconds)
 	VMSends                             = stats.Int64("vm/sends", "Counter for sends processed by the VM", stats.UnitDimensionless)
 	VMApplied                           = stats.Int64("vm/applied", "Counter for messages (including internal messages) processed by the VM", stats.UnitDimensionless)
+	VMExecutionWaiting                  = stats.Int64("vm/execution_waiting", "Counter for VM executions waiting to be assigned to a lane", stats.UnitDimensionless)
+	VMExecutionActive                   = stats.Int64("vm/execution_default", "Counter for active VM executions", stats.UnitDimensionless)
```

Comment:
nit: can we call this "running" (it's more specific than "active"). Also, wrong metric name.

```suggestion
	VMExecutionRunning                   = stats.Int64("vm/execution_running", "Counter for VM executions currently running", stats.UnitDimensionless)
```